src/main.ts

```diff
@@ -0,0 +1,66 @@
+import { extractOutfileFromRunArguments } from './utils'
+import { Dependencies } from './dependencies'
+import { EnvironmentVariables, Inputs } from './types'
+
+const INTERNAL_OUTFILE = 'SalesforceCodeAnalyzerResults.json'
+
+export const MESSAGES = {
+    MISSING_NORMALIZE_SEVERITY: 'Missing required --normalize-severity option from run-arguments input.'
+}
+
+/**
+ * The main function for the action.
+ * @returns {Promise<void>} Resolves when the action is complete.
+ */
+export async function run(dependencies: Dependencies): Promise<void> {
+    try {
+        dependencies.startGroup('Preparing Environment')
+        // TODO: NICE TO HAVES:
+        // * Verify that Salesforce CLI "sf" is installed
+        // * Verify that sfdx-scanner plugin is installed (and if not, then install it as a separate step)
+        // * Echo version of sfdx-scanner in use
```

Comment:
I hope to do these along with the output analysis in the next work item. And I might break it up into 2 separate PRs to make things easy.